apps/sensenet/src/context/ThemeContext.tsx

```diff
@@ -1,5 +1,4 @@
 import createMuiTheme, { Theme } from '@material-ui/core/styles/createMuiTheme'
 import React from 'react'
-import theme from '../components/theme'
 
-export const ThemeContext = React.createContext<Theme>(createMuiTheme(theme))
+export const ThemeContext = React.createContext<Theme>(createMuiTheme(undefined))
```

Comment:
I cannot cause any problem cause we override the default style where we need. But if I changed it to undefined I got errors everywhere where I would like to use theme (theme is maybe undefined).
So I should be the defaukt theme instead of undefined! It is still better.

Reply:
I disagree with you about this one. Because of this we miss a bug that  ResponsiveContextProvider is not a child of ThemeProvider altough it uses useTheme it will never get the values we override only the default ones from material ui. Rewriting the useTheme hook as I suggested also gets rid of the errors you mentioned.